components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -619,10 +619,10 @@ void MAMMicrophysics::initialize_impl(const RunType run_type) {
   const int curr_month = start_of_step_ts().get_month() - 1;  // 0-based
   if (config_.linoz.compute) {
     scream::mam_coupling::update_tracer_data_from_file(
-      LinozDataReader_, curr_month, *LinozHorizInterp_, linoz_data_);
+      LinozDataReader_, curr_month+linoz_data_.offset_time_index_, *LinozHorizInterp_, linoz_data_);
   }
   scream::mam_coupling::update_tracer_data_from_file(
-      TracerDataReader_, curr_month, *TracerHorizInterp_, tracer_data_);
+      TracerDataReader_, curr_month+tracer_data_.offset_time_index_, *TracerHorizInterp_, tracer_data_);
```

Comment:
Should a similar change also be made for the call interface at line 629?

Reply:
@odiazib: We eventually want to support multi-year files for all input data. Do you think we should consider adding this fix to the elevated emissions, as Kai mentioned above?

Reply:
Sounds good.